### PR TITLE
{lib}[foss/2018b] deepTools v3.3.1 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+easyblock = 'Bundle'
```
Comment (boegel) [inline easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb]: @verdurin You should use `PythonBundle` here (tests trip over that now)

File: easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('Pysam', '0.15.1', versionsuffix),
+    ('matplotlib', '3.0.0', versionsuffix),
+]
+
+exts_defaultclass = 'PythonPackage'
+exts_filter = ("python -c 'import %(ext_name)s'", '')
```
Comment (boegel) [inline easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb]: both lines can be removed when using `PythonBundle`

File: easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+sanity_check_paths = {
+    'files': ['bin/bamCompare', 'bin/bamCoverage', 'bin/bamPEFragmentSize', 'bin/computeGCBias', 'bin/computeMatrix',
+              'bin/correctGCBias', 'bin/multiBamSummary', 'bin/plotCorrelation', 'bin/plotCoverage',
+              'bin/plotHeatmap', 'bin/plotProfile'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = ["python -c 'import deeptools'"]
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
```
Comment (boegel) [inline easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb]: You can remove this when using `PythonBundle`

File: easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+]
+
+sanity_check_paths = {
+    'files': ['bin/bamCompare', 'bin/bamCoverage', 'bin/bamPEFragmentSize', 'bin/computeGCBias', 'bin/computeMatrix',
+              'bin/correctGCBias', 'bin/multiBamSummary', 'bin/plotCorrelation', 'bin/plotCoverage',
+              'bin/plotHeatmap', 'bin/plotProfile'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = ["python -c 'import deeptools'"]
```
Comment (boegel) [inline easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb]: The `import` test is already done for the `deepTools` extension (but it doesn't hurt to check again, I guess?)

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in this PR)
rescomp2 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1aa986d96022de6f3326ce61222f795a for a full test report.

File: easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb]: Done.

File: easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb (hunk 2 above)
Comment (verdurin) [reply easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb]: Done.

File: easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb (hunk 3 above)
Comment (verdurin) [reply easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb]: Done.

File: easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb (hunk 4 above)
Comment (verdurin) [reply easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb]: Also done.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in this PR)
rescomp2 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/60a33a55cebdb02ab7a8a5608428c332 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
rescomp2 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/10b566284ab0e0552e1e1edf5c293844 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/1cd60d6da498fc340993c0a92a9c8998 for a full test report.

File: easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+]
+
+sanity_check_paths = {
+    'files': ['bin/bamCompare', 'bin/bamCoverage', 'bin/bamPEFragmentSize', 'bin/computeGCBias', 'bin/computeMatrix',
+              'bin/correctGCBias', 'bin/multiBamSummary', 'bin/plotCorrelation', 'bin/plotCoverage',
+              'bin/plotHeatmap', 'bin/plotProfile'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+moduleclass = 'lib'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb]: `bio`?

Comment (verdurin) [reply easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb]: Wasn't sure about this myself. The previous version was `lib`, but I'm happy to change if you prefer `bio`.

Comment (verdurin) [reply easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb]: Fixed.

**Comment by migueldiascosta** [comment]
Going in, thanks @verdurin!

**Comment by migueldiascosta** [comment]
Going in, thanks @verdurin!

**Pull request merged**